pkg/metrics/metrics.go

```diff
@@ -117,6 +127,11 @@ var (
 		Name:      "cluster_verification_failed",
 		Help:      "Failed on the cluster upgrade verification step",
 	}, []string{nameLabel})
+	metricUpgradeWindowBreached = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Need to register this new metric to the init func below.

Reply:
:facepalm: great catch, thanks. Fixed in commit 16fe7ac